root/usr/bin/redsift/protocol.js

```diff
@@ -19,22 +14,31 @@ function flattenNestedArrays(value) {
   return [value];
 }
 
+function getFileContent(d) {
+  return fs.readFileSync(
```

Comment:
Ideally we can speed up the read here by doing an array of `readFile` and resolve their promises. But I see that as a future improvement.